src/main/java/com/zufar/onlinestore/cart/api/ItemsTotalPriceCalculator.java

```diff
@@ -0,0 +1,20 @@
+package com.zufar.onlinestore.cart.api;
+
+import com.zufar.onlinestore.cart.dto.ShoppingSessionItemDto;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.stereotype.Service;
+
+import java.math.BigDecimal;
+import java.util.Collection;
+
+@Slf4j
+@Service
+public class ItemsTotalPriceCalculator {
+
+    public BigDecimal calculate(Collection<ShoppingSessionItemDto> items) {
```

Comment:
will this method be called in other up-tier services? If it isn't I think there's no reason to use so width interface for collection here

Reply:
Done